libs/ui/packages/native/src/components/Layout/ScrollListContainer/index.tsx

```diff
@@ -17,21 +17,22 @@ type ScrollListContainerProps = BaseStyledProps &
  ** This Layout is a wrapper for FlatList that accepts
  ** complex onScroll callback for react-native-reanimated.
  */
-const ScrollListContainer = ({
-  children,
-  onScroll,
-  horizontal = false,
-  ...props
-}: ScrollListContainerProps): JSX.Element => {
-  return (
-    <AnimatedFlatList
-      data={[...React.Children.toArray(children)]}
-      renderItem={({ item }: { item: React.ReactNode }) => <View>{item}</View>}
-      onScroll={onScroll}
-      horizontal={horizontal}
-      {...props}
-    />
-  );
-};
+const ScrollListContainer = forwardRef(
+  (
+    { children, onScroll, horizontal = false, ...props }: ScrollListContainerProps,
+    ref,
+  ): JSX.Element => {
```

Comment:
And we should be able to remove this one too because forwardRef type it correctly
```suggestion
  ) => {
```